services/fdb/fdb.proto

```diff
@@ -360,6 +553,9 @@ message FDBCLITransaction {
 
 message FDBCLIUnknownAction {}
 
+// FDBCLIRequest defines the fdbcli command(s) to run.
+// This will replicate fdbcli --exec "command(s)" (where N commands are
```

Comment:
does --exec foo;bar provide some transactional/sematic difference vs 'fdb foo ; fdb bar' (e.g. would it be different than running them in a shell script sequentially?)